pkg/ctl/delete/cluster.go

```diff
@@ -103,11 +104,23 @@ func doDeleteCluster(rc *cmdutils.ResourceCmd) error {
 	}
 
 	{
+
+		logger.Info("cleaning up LoadBalancer services")
+		if err := ctl.GetCredentials(cfg); err != nil {
+			return err
+		}
+		client, err := ctl.NewClient(cfg, false)
```

Comment:
```suggestion
		clientSet, err := ctl.NewClientSet(cfg, false)
```

Reply:
I guess you meant `ctl.NewStdClientSet`, correct?

Reply:
No, that one goes with embedded token. Please clarify.

Reply:
Yes, that's right, it'd be `ctl.NewStdClientSet`.

Reply:
What's the difference between embedding a token or not? Right now the client is created without one, following what `eksctl utils write-kubecobfig` does, whereas `NewNewStdClientSet()` doesn't.

Reply:
Good question!

The client with embedded token is most desirable for internal use. We don't need to execute the authenticator plugin and we instead call the authenticator as a library and store the token in the client config structure.

When we want to write the `kubeconfig` file out for users to consume, we want it to use the authenticator binary every time user calls `kubectl`. If we were to use embedded token there, we'd have to provide a token refresh mechanism also.

Reply:
Ah, great, thanks a lot for the clarification